src/main/java/org/folio/util/InventoryUtils.java

```diff
@@ -0,0 +1,61 @@
+package org.folio.util;
+
+import io.vertx.core.json.JsonObject;
+import lombok.extern.log4j.Log4j2;
+import org.folio.rest.jaxrs.model.Contributor;
+
+import java.util.List;
+
+@Log4j2
+public class InventoryUtils {
+
+  public static final String INSTANCE_ID = "id";
```

Comment:
We can create enums for storing field names

Reply:
> We can create enums for storing field names

Good one, can be added with POL product id changes that will be scheduled this sprint